lib/hope-tab-list.js

```diff
@@ -27,6 +27,13 @@ fxosComponent.register('hope-tab-list', {
 
     // Events
     scheduler.attachDirect(this.els.inner, 'scroll', this.onScroll);
+    this.els.inner.addEventListener('scrollend', (evt) => {
+      // We detach the direct block to flush the scheduler queue then re-attach it
+      scheduler.detachDirect(this.els.inner, 'scroll', this.onScroll);
+      setTimeout(() => {
+        scheduler.attachDirect(this.els.inner, 'scroll', this.onScroll);
+      });
+    });
```

Comment:
I'm going back and forth on this one.

Basically the scheduler protects interaction with a "timeout window". If we don't get a new event for 350ms we stop protecting and flush.
But if the direct block is detached we flush right away.
This works very well for cases where you attach a direct block on touchmove when the touchstart comes in, and detach it when the touchend comes.

I'm trying to reproduce the same but it's a bit awkward because we don't have a scrollstart event.